### PR TITLE
Versions update

### PR DESCRIPTION
**Pull request by koperagen**

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 kotlinJupyter = "0.11.0-358"
 ktlint = "3.4.5"
-kotlin = "1.8.20"
+kotlin = "1.9.0"
 dokka = "1.8.10"
 libsPublisher = "0.0.60-dev-30"
 # "Bootstrap" version of the dataframe, used in the build itself to generate @DataSchema APIs,
 # dogfood Gradle / KSP plugins in tests and idea-examples modules
-dataframe = "0.11.0"
+dataframe = "0.12.0-dev-2255"
```
Comment (zaleslaw) [inline gradle/libs.versions.toml:10]: why the exact version  is specified?

File: gradle/libs.versions.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 [versions]
-ksp = "1.8.20-1.0.11"
+ksp = "1.9.0-1.0.12"
 kotlinJupyter = "0.11.0-358"
 ktlint = "3.4.5"
-kotlin = "1.8.20"
+kotlin = "1.9.0"
```
Comment (zaleslaw) [inline gradle/libs.versions.toml:5]: Did you test, does it compile well?

File: gradle/libs.versions.toml (hunk 1 above)
Comment (koperagen) [reply gradle/libs.versions.toml:10]: This bootstrap version of dataframe is built with KSP 1.9.0-1.0.12 that is needed to work with Kotlin 1.9.0

File: gradle/libs.versions.toml (hunk 2 above)
Comment (koperagen) [reply gradle/libs.versions.toml:5]: Actually, no :) Something's broken in gradle-plugin module

File: README.md [hunk 3]
```diff
@@ -4,6 +4,7 @@
 [![Kotlin](https://img.shields.io/badge/kotlin-1.8.20-blue.svg?logo=kotlin)](http://kotlinlang.org)
 [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlinx/dataframe?color=blue&label=Maven%20Central)](https://search.maven.org/artifact/org.jetbrains.kotlinx/dataframe)
 [![GitHub License](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Kotlin/dataframe/HEAD)
```
Comment (Jolanrensen) [inline README.md:7]: yay, binder!

**Review by Jolanrensen — APPROVED** [review]

File: gradle/libs.versions.toml (hunk 2 above)
Comment (koperagen) [reply gradle/libs.versions.toml:5]: last commit resolved the issue

**Pull request merged**